root/static/scripts/edit/URLCleanup.js

```diff
@@ -3453,6 +3454,12 @@ const CLEANUPS: CleanupEntries = {
               result: prefix === 'venue',
               target: ERROR_TARGETS.RELATIONSHIP,
             };
+          case LINK_TYPES.otherdatabases.recording:
+            return {
+              error: l('Only RYM music videos can be linked to recordings.'),
```

Comment:
Is it really necessary to have a special message here? Other RYM links were already denied for linking to recordings.

Reply:
I think it might be useful in case people try to link a single to a recording or something? :)